addons/utm/views/utm_campaign_views.xml

```diff
@@ -9,12 +9,18 @@
                 <field name="tag_ids"/>
                 <field name="user_id"/>
                 <field name="is_auto_campaign"/>
+                <filter string="My Campaigns" name="filter_assigned_to_me"
+                            domain="[('user_id', '=', uid)]"
```

Comment:
Strange spacing, please use the same spacing as the "Tags" group below when going on a new line.